src/encoded/audit/experiment.py

```diff
@@ -2906,6 +2906,25 @@ def audit_library_RNA_size_range(value, system, excluded_types):
     return
 
 
+def audit_RNA_library_RIN(value, system, excluded_types):
+    '''
+    An RNA library should have a RIN specified.
+    '''
+
```

Comment:
Can we collapse this all onto one line?  
```
RNAs = ['RNA', 'polyadenylated mRNA', 'miRNA']
for rep in value['replicates']:
```
It's a pretty short list.  If there is a concern about needing to add elements to it in the future then
```
RNAs = [
    'RNA',
    'polyadenylated mRNA',
    'miRNA',
]
for rep in value['replicates']:
```
would be prefered.

Reply:
fixed